java-checks/src/test/files/checks/UndocumentedApiCheck/UndocumentedApi.java

```diff
@@ -94,20 +94,20 @@ protected void doSomething() {                 // Compliant - not public
    * Valid descriptions.
```

Comment:
Probably yes.

Reply:
In fact, we should ideally move the files to the compiled module, but it's quite a big change, I'm afraid it will pollute the diff of this PR which is already non-trivial. I suggest to not move the files in this PR.